scipy/stats/_morestats.py

```diff
@@ -3741,14 +3741,31 @@ def circmean(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
 
     Examples
     --------
-    >>> import numpy as np
-    >>> from scipy.stats import circmean
-    >>> circmean([0.1, 2*np.pi+0.2, 6*np.pi+0.3])
-    0.2
+    For simplicity, all angles are printed out in degrees.
 
+    >>> import numpy as np
     >>> from scipy.stats import circmean
-    >>> circmean([0.2, 1.4, 2.6], high = 1, low = 0)
-    0.4
+    >>> import matplotlib.pyplot as plt
+    >>> angles=np.deg2rad(np.array([20, 30, 330]))
+    >>> circmean=circmean(angles)
+    >>> np.rad2deg(circmean)
+    7.294976657784009
+
+    >>> mean=angles.mean()
+    >>> np.rad2deg(mean)
+    126.66666666666666
+
+    Plot and compare the results of *circmean* and the regular *mean*.
```

Comment:
Asterisks are emphasis. How about:
```suggestion
    Plot and compare the circular mean against the arithmetic mean.
```